IDEAS/Buildings/Components/Interfaces/PartialZone.mo

```diff
@@ -3,6 +3,10 @@ partial model PartialZone "Partial model for thermal building zones"
   replaceable package Medium =
     Modelica.Media.Interfaces.PartialMedium "Medium in the component"
       annotation (choicesAllMatching = true);
+
+  parameter Modelica.Fluid.Types.Dynamics energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial
+    "Type of energy balance: dynamic (3 initialization options) or steady state";
+  parameter Real corrCV = 5 "Correction factor for thermal capacity of zone air.";
```

Comment:
fine for me, but this will create backward incompatibiliy. But I gess, for version 1.0 everything can :)